### DISCUSSION TITLE
result_summary.tsv

### DISCUSSION THREAD

**Discussion by TCHeaven** [hahnlab/CAFE5#186] [Q&A]
Forgive me, I dont understand the result_summary.tsv file, if I count the number of FamliyIDs that have +ve and -ve change values - or if i sum the change or count columns - this does not match the counts given in model_clade_results.txt, also change and/or count of zero is given from some FamilyIDs. What data is being presented in this file?

**Comment by benfulton** [comment]
The CAFE application doesn't create a file called result_summary.tsv. Are you looking at one of the auxiliary scripts? If so, which one?
